### PR TITLE
Merge ObstacleStatus and RobotStatus

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: src/isar_robot/robotinterface.py [hunk 1]
```diff
@@ -329,5 +333,5 @@ def _update_pressure_level(self) -> float:
         return self.pressure_level
 
     def _update_obstacle_status(self) -> bool:
-        self.obstacle_status = False
+        self.obstacle_status = True
```
Comment (oysand) [inline src/isar_robot/robotinterface.py:336]: This should be configurable and not hard coded. I don't think we want the default to be True.

**Review by oysand — CHANGES_REQUESTED** [review]
See comments

File: src/isar_robot/robotinterface.py (hunk 1 above)
Comment (oysand) [reply src/isar_robot/robotinterface.py:336]: We could also keep it only set in the constructor, but it should not be overwritten like this in the code. Suggest to remove this line entirely

Comment (UsamaEquinorAFK) [reply src/isar_robot/robotinterface.py:336]: We keep the default false yes. It is set true for test purposes

**Pull request merged**